tests/unit/helpers/extend-logger.test.js

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable no-console */
+import winston from 'winston';
+import extendLogger from '../../../source/helpers/extend-logger';
+
+describe('Helper - Function extend-debug-logger', () => {
```

Comment:
The function name in the test suite name should match the name in real code. In this case: `extendLogger`